sstmac/sst_core/integrated_core.cc

```diff
@@ -100,7 +100,7 @@ int_vector_from_py_array(PyObject* tuple, std::vector<int>& vec)
   }
 }
 
-void
+bool
```

Comment:
Aside from the dead write below I don't think this return value is ever actually used?

Reply:
removed